source/reference/cutover-process.txt

```diff
@@ -63,19 +88,67 @@ Steps
       migration, you must issue a commit request for each ``mongosync``
       instance.
 
+      Request
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/requests/commit.sh
+         :language: shell
+
+      Response
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/responses/success.json
+         :language: json
+         :copyable: false
+
       .. note:: 
 
-         Ensure that the ``mongosync`` process response for the
-         ``commit`` request indicates that the ``mongosync`` state is
-         ``COMMITTING`` or ``COMMITTED``, which is the correct
-         ``mongosync`` state when you send a ``commit`` request.
+         After you submit a ``commit`` request, call the ``progress`` endpoint
+         to ensure that the ``mongosync`` state is ``COMMITTING`` or 
+         ``COMMITTED``.
```

Comment:
[Suggestion]

Since we tell users to call the `progress` endpoint in the following step, what if we remove this note and move this information to the introduction of the next step. As in, the introduction to the next step would be something like:

```
Call the ``progress`` endpoint and confirm that the response contains
the following fields:

- ``state`` is  ``COMMITTING`` or ``COMMITTED``.
- ``canWrite`` is ``true``. If ``canWrite`` is ``false``, wait until
  ``progress`` shows ``canWrite`` is ``true``.

Reply:
Thanks for the suggestion! I like this idea but I think since the note is not an essential step (and also part of the commit step) I think I will leave as is for now.